### ISSUE TITLE
Acalculia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10207]
**Preferred term label:**
Acalculia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C5551529

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
GERSTMANN SYNDROME

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
See [Dyscalculia HP:0002442](https://hpo.jax.org/app/browse/term/HP:0002442). Similar question was with issue #10208

**Comment by MickeySegal** [comment]
I agree that acalculia and dyscalculia on a continuum, so I combined our terms and will use the dyscalculia name. For Gerstmann syndrome, each terms seems to be used with about equal frequency.

Also note that for https://hpo.jax.org/app/browse/term/HP:0002442 neither UMLS term is on MedGen; instead, https://www.ncbi.nlm.nih.gov/medgen/C0869474 and https://www.ncbi.nlm.nih.gov/medgen/C5551529 are.

**Comment by pnrobinson** [comment]
I will add Acalculia as an exact synonym to dyscalculia for now. I do not think there is agreement in the community as to what these words represent. We can revisit if we can recruit experts to advise later on.
